ascent_macro/src/ascent_codegen.rs

```diff
@@ -320,7 +326,8 @@ pub(crate) fn compile_mir(mir: &AscentMir, is_ascent_run: bool) -> proc_macro2::
       res
    } else {
       quote! {
-         {
+         {{
+            #![allow(clippy::all)]
```

Comment:
Could this be combined with the `allow` attribute on line 333?

Reply:
Done.